Tests/TuistGeneratorTests/Graph/Mocks/MockGraphLoaderCache.swift

```diff
@@ -3,6 +3,14 @@ import Foundation
 @testable import TuistGenerator
 
 final class MockGraphLoaderCache: GraphLoaderCaching {
+    var packageNodes: [AbsolutePath: PackageNode] = [:]
+
+    func package(_: AbsolutePath) -> PackageNode? {
```

Comment:
I'd add an implementation to those methods, like we do with the other methods that are part of this mock.